canonical/ubuntu.com
# WD-11971 - add new tab to /download/risc-v

## Done

DO NOT MERGE UNTIL Jul 9th
Added new tab to /download/risc-v

## QA

- [demo link](https://ubuntu-com-14016.demos.haus/download/risc-v)
- [copy doc](https://docs.google.com/document/d/1kkJeq9HdH3iWV35HwihcJd3CxZw9Jy21W66lJl8VbWs/edit)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the section is correct according to copy doc

## Issue / Card
[WD-11971](https://warthogs.atlassian.net/browse/WD-11971)

[WD-11971]: https://warthogs.atlassian.net/browse/WD-11971?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
